lighthouse-core/audits/screenshot-thumbnails.js

```diff
@@ -0,0 +1,119 @@
+      description: 'Screenshot Thumbnails',
+      helpText: 'This is what the load of your site looked like.',
+      requiredArtifacts: ['traces']
+    };
+  }
+
+  /**
+   * Scales down an image to THUMBNAIL_WIDTH using nearest neighbor for speed, maintains aspect
+   * ratio of the original thumbnail.
+   *
+   * @param {{width: number, height: number, data: !Array<number>}} imageData
+   * @return {{width: number, height: number, data: !Array<number>}}
+   */
+  static scaleImageToThumbnail(imageData) {
+    const scaledWidth = THUMBNAIL_WIDTH;
+    const scaleFactor = imageData.width / scaledWidth;
+    const scaledHeight = Math.floor(imageData.height / scaleFactor);
+
+    const outPixels = new Uint8Array(scaledWidth * scaledHeight * 4);
+
+    for (let i = 0; i < scaledWidth; i++) {
+      for (let j = 0; j < scaledHeight; j++) {
+        const origX = Math.floor(i * scaleFactor);
+        const origY = Math.floor(j * scaleFactor);
+
+        const origPos = (origY * imageData.width + origX) * 4;
+        const outPos = (j * scaledWidth + i) * 4;
+
+        outPixels[outPos] = imageData.data[origPos];
+        outPixels[outPos + 1] = imageData.data[origPos + 1];
+        outPixels[outPos + 2] = imageData.data[origPos + 2];
+        outPixels[outPos + 3] = imageData.data[origPos + 3];
+      }
+    }
+
+    return {
+      width: scaledWidth,
+      height: scaledHeight,
+      data: outPixels,
+    };
+  }
+
+  /**
+   * @param {!Artifacts} artifacts
+   * @return {!AuditResult}
+   */
+  static audit(artifacts) {
+    const trace = artifacts.traces[Audit.DEFAULT_PASS];
+    const cachedThumbnails = new Map();
+
+    return artifacts.requestSpeedline(trace).then(speedline => {
+      const thumbnails = [];
+      const analyzedFrames = speedline.frames.filter(frame => !frame.isProgressInterpolated());
+
+      for (let i = 1; i <= NUMBER_OF_THUMBNAILS; i++) {
+        const targetTimestamp = speedline.beginning + speedline.complete * i / NUMBER_OF_THUMBNAILS;
+
+        let frameForTimestamp = null;
+        if (i === NUMBER_OF_THUMBNAILS) {
+          frameForTimestamp = analyzedFrames[analyzedFrames.length - 1];
+        } else {
+          analyzedFrames.forEach(frame => {
+            if (frame.getTimeStamp() <= targetTimestamp) {
+              frameForTimestamp = frame;
+            }
+          });
+        }
+
+        const imageData = frameForTimestamp.getParsedImage();
+        const thumbnailImageData = ScreenshotThumbnails.scaleImageToThumbnail(imageData);
+        const base64Data = cachedThumbnails.get(frameForTimestamp) ||
+            jpeg.encode(thumbnailImageData, 90).data.toString('base64');
+
+        cachedThumbnails.set(frameForTimestamp, base64Data);
+        thumbnails.push({
+          timing: Math.round(targetTimestamp - speedline.beginning),
+          timestamp: targetTimestamp * 1000,
+          data: base64Data,
+        });
+      }
+
+      return {rawValue: thumbnails};
```

Comment:
I wasn't a fan of those options originally since `details` maps to the details renderer and, again, rendering info not related to how to display the audit itself just its collapsible extraneous information. In the generic view, "rawValue" as it's the raw screenshot data not formatted for a renderer, made the most sense to me for third parties consuming the JSON, and makes it clear that a generic renderer does not support this. I'll mark as informative and explicitly set the score so rawValue isn't used there for sure, but I'd want to avoid sticking into details as a list or table when it's distinctly something *not* handled or treated as either of those.